### ISSUE TITLE
Windows not closing

### ISSUE THREAD

**Issue by Beeefin** [ultravnc/UltraVNC#88]
Not sure if this is an issue that I only experience. However, when I am remoting to Windows PC's whenever I try minimizing/closing a window it won't physically close. The windows stays on-screen until I click the Windows start button and it seems like it refreshes the display or something. Wondering if anyone else seems to have this issue, or if it is just me.

**Comment by RudiDeVos** [comment]
A standard windows application is not closing or some traces are left on the display ?
What capture method are you using, OS

**Comment by Beeefin** [comment]
It's any window open within Windows 10, (file explorer, edge, or any program that pulls up a window. It only happens sometimes, I can record a video or picuree if that would help.

**Comment by Neustradamus** [comment]
@Beeefin: Have you done videos?
If yes, please share it!

**Comment by Neustradamus** [comment]
@Beeefin: Can you publish your videos?

**Comment by Neustradamus** [comment]
@Beeefin: Can you reply on your ticket?

**Comment by Neustradamus** [comment]
@Beeefin: It has been solved?

**Comment by Beeefin** [comment]
Yeah I just couldn't get it to reproduce consistently anymore. Might have been something with autoCAD - not sure. Felt like my PC's with AutoCAD had it the most.
